oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2BatchDAO.java

```diff
@@ -81,6 +92,36 @@ public void flush(List<PrepareRequest> prepareRequests) {
         }
     }
 
+    private void executeSql(List<PrepareRequest> prepareRequests, int maxBatchSqlSize) {
+        if (log.isDebugEnabled()) {
+            log.debug("execute sql batch. data size:{}", prepareRequests.size());
+        }
+        Map<String, List<PrepareRequest>> batchRequestMap = new HashMap<>();
+        for (PrepareRequest prepareRequest : prepareRequests) {
+            SQLExecutor sqlExecutor = (SQLExecutor) prepareRequest;
+            if (batchRequestMap.get(sqlExecutor.getSql()) == null) {
```

Comment:
When this could be null?

Reply:
in this batch prepareRequests when first same name sql to insert batchRequestMap. this get will be null.
i modify this by  sonatype-lift review advise. modify after:
            if (batchRequestMap.containsKey(sqlExecutor.getSql())) {
                batchRequestMap.get(sqlExecutor.getSql()).add(prepareRequest);
            } else {
                List<PrepareRequest> prepareRequestList = new ArrayList<>();
                batchRequestMap.put(sqlExecutor.getSql(), prepareRequestList);
            }